src/drivers/intel/hda/hda-dma.c

```diff
@@ -622,15 +616,16 @@ static int hda_dma_stop(struct dma_chan_data *channel)
 		hda_dma_host_stop(channel);
```

Comment:
looking through this - is the call to `pm_runtime_put(PM_RUNTIME_HOST_DMA_L1, 0);` in `hda_dma_host_start()` actually correct?